aspnetcore/fundamentals/servers/httpsys.md

```diff
@@ -0,0 +1,146 @@
+---
+title: HttpSys web server implementation in ASP.NET Core
+author: rick-anderson
+description: Introduces HttpSys, a web server for ASP.NET Core on Windows. Built on the Http.Sys kernel mode driver, HttpSys is an alternative to Kestrel that can be used for direct connection to the Internet without IIS.
+keywords: ASP.NET Core, HttpSys, HttpListener, url prefixes, SSL 
+ms.author: riande
+manager: wpickett
+ms.date: 08/07/2017
+ms.topic: article
+ms.assetid: 0a7286e4-6428-424e-b5c4-5c98815cf61c
+ms.technology: aspnet
+ms.prod: asp.net-core
+uid: fundamentals/servers/HttpSys
+---
+# HttpSys web server implementation in ASP.NET Core
+
+By [Tom Dykstra](http://github.com/tdykstra) and [Chris Ross](https://github.com/Tratcher)
+
+> [!NOTE]
+> This topic applies only to ASP.NET Core 2.0 and later. In earlier versions of ASP.NET Core, HttpSys is named [WebListener](WebListener.md).
+
+HttpSys is a [web server for ASP.NET Core](index.md) that runs only on Windows. It's built on the [Http.Sys kernel mode driver](https://msdn.microsoft.com/library/windows/desktop/aa364510.aspx). HttpSys is an alternative to [Kestrel](kestrel.md) that offers some features that Kestel doesn't. **HttpSys can't be used with IIS or IIS Express, as it isn't compatible with the [ASP.NET Core Module](aspnet-core-module.md).**
+
+HttpSys supports the following features:
+
+- [Windows Authentication](xref:security/authentication/windowsauth)
+- Port sharing
+- HTTPS with SNI
+- HTTP/2 over TLS (Windows 10)
+- Direct file transmission
+- Response caching
+- WebSockets (Windows 8)
+
+Supported Windows versions:
+
+- Windows 7 and Windows Server 2008 R2 and later
+
+[View or download sample code](https://github.com/aspnet/Docs/tree/master/aspnetcore/fundamentals/servers/httpsys/sample)
+
+## When to use HttpSys
+
+HttpSys is useful for deployments where you need to expose the server directly to the Internet without using IIS.
+
+![HttpSys to Internet](httpsys/_static/httpsys-to-internet.png)
+
+Because it's built on Http.Sys, HttpSys doesn't require a reverse proxy server for protection against attacks. Http.Sys is mature technology that protects against many kinds of attacks and provides the robustness, security, and scalability of a full-featured web server. IIS itself runs as an HTTP listener on top of Http.Sys. 
+
+HttpSys is also a good choice for internal deployments when you need one of the features it offers that you can't get by using Kestrel.
```

Comment:
about the same but a few less words.
HttpSys is a good choice for internal deployments when you need one of its features not available in Kestrel.

or even

HttpSys is a good choice for internal deployments when you need a feature not available in Kestrel.